web/Views/Landlord/Profile.cshtml

```diff
@@ -72,4 +91,8 @@
         <p>Charter status: Pending</p>
     }
     
+    @using (Html.BeginForm("GetInviteLink", "Admin", new { landlordId = Model.LandlordId }, FormMethod.Post))

```

Comment:
Can you make sure this is only visible to admins?

Reply:
ok done